main.py

```diff
@@ -1,14 +1,18 @@
-import requests
-bestMovies =[]
-for i in range(1,26):
+from requests import Session
+
+sess = Session()

```

Comment:
always better to use Sessions, specially if we're making multiple http requests to the same host.